cve_bin_tool/cvedb.py

```diff
@@ -289,9 +304,9 @@ def populate_db(self):
         insert_severity = """
         INSERT or REPLACE INTO cve_severity(
             CVE_number,
-            severity, 
-            description, 
-            score, 
+            severity,
+            description,
+            score,
```

Comment:
Nice cleanup here.  I'll bet that was my fault -- I'm horrible about spaces at end of lines and need to fix my vim settings so it warns me about them again.